source/developer/slash-commands.rst

```diff
@@ -151,18 +151,18 @@ Messages with advanced formatting can be created by including an :doc:`attachmen
   
-  Similarly, `Enable integrations to override profile picture icons <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-profile-picture-icons>`__ must be set to ``true`` in ``config.json`` to override usernames. Enable them from **System Console > Integrations > Integration Management** or ask your Mattermost System Admin. If not enabled, the icon of the creator of the webhook URL is used to post messages.
+  Similarly, `Enable integrations to override profile picture icons <https://docs.mattermost.com/configure/configuration-settings.html#enable-integrations-to-override-profile-picture-icons>`__ must be set to ``true`` in ``config.json`` to override usernames. Enable them from **System Console > Integrations > Integration Management** or ask your Mattermost System Admin. If not enabled, the icon of the creator of the webhook URL is used to post messages.
 
 Tips and Best Practices
 ------------------------
 
 1. Slash commands are designed to easily allow you to post messages. For other actions such as channel creation, you must also use the `Mattermost APIs <https://api.mattermost.com>`__.
 
-2. Posts size is limited to 16393 characters for servers running `Mattermost Server v5.0 or later <https://docs.mattermost.com/administration/important-upgrade-notes.html>`__. Use the `extra_responses <https://developers.mattermost.com/integrate/slash-commands/#parameters>`__ field to reply to a triggered slash command with more than one post.
+2. Posts size is limited to 16393 characters for servers running `Mattermost Server v5.0 or later <https://docs.mattermost.com/upgrade/important-upgrade-notes.html>`__. Use the `extra_responses <https://developers.mattermost.com/integrate/slash-commands/#parameters>`__ field to reply to a triggered slash command with more than one post.
 
-3. You can restrict who can create slash commands in `System Console > Integrations > Integration Management <https://docs.mattermost.com/administration/config-settings.html#restrict-managing-integrations-to-admins>`__.
+3. You can restrict who can create slash commands in `System Console > Integrations > Integration Management <https://docs.mattermost.com/configure/configuration-settings.html#restrict-managing-integrations-to-admins>`__.
 
 4. Mattermost outgoing webhooks are Slack-compatible. You can copy-and-paste code used for a Slack outgoing webhook to create Mattermost integrations. Mattermost `automatically translates Slack's JSON format <https://docs.mattermost.com/developer/slash-commands.html?highlight=translate%20slack%20data%20format%20mattermost#translate-slack-s-data-format-to-mattermost>`__.
```

Comment:
Please update this link to remove the highlighting as follows: ``https://docs.mattermost.com/developer/slash-commands.html#translate-slack-s-data-format-to-mattermost``

Reply:
I did it with a seperate PR as VS code is crashing in my computer somehow

Reply:
This link has changed to: https://developers.mattermost.com/integrate/admin-guide/admin-slash-commands/#translate-slacks-data-format-to-mattermost. @Sayanta66, could you provide the PR link where this change was made so we can resolve this conversation and ensure the change is confirmed? Thank you!

Reply:
@justinegeffen #4859 this one